src/common/BaseAnnotator.ts

```diff
@@ -151,4 +152,12 @@ export default class BaseAnnotator extends EventEmitter {
         this.store.dispatch<any>(store.fetchAnnotationsAction()); // eslint-disable-line @typescript-eslint/no-explicit-any
         this.store.dispatch<any>(store.fetchCollaboratorsAction()); // eslint-disable-line @typescript-eslint/no-explicit-any
     }
+
+    protected handleInitialized(status = false): void {
+        const isInitialized = areAnnotationsInitialized(this.store.getState());
```

Comment:
Should we handle this in the `eventing/init` module to simplify the annotator? Basically, if `isInitialized` was false but now is true, emit?

Reply:
I think it is more explicit here and also saves on a dispatch for every call to `init`

Reply:
It's also kind of confusing, since calling this method with `false` doesn't do anything. Why allow the status to be passed in as an argument? We're trying to control the behavior of the event, not the set, right?